packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts

```diff
@@ -60,6 +55,41 @@ export class EditorSettingsSidebarComponent {
 		this.editor = editor;
 	}
 
+	//#region Generic methods
+
+	/**
+	 * Clicks a button matching the accessible name.
+	 *
+	 * @param {string} name Accessible name of the button.
+	 */
+	async clickButton( name: string ): Promise< void > {
```

Comment:
The trend for now, at least for me, has been to implement more generic methods instead of tailored flows.